lacework/resource_lacework_data_export_rule.go

```diff
@@ -0,0 +1,201 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkDataExportRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkDataExportRuleCreate,
+		Read:   resourceLaceworkDataExportRuleRead,
+		Update: resourceLaceworkDataExportRuleUpdate,
+		Delete: resourceLaceworkDataExportRuleDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkDataExportRule,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "The name of the data export rule",
+				Required:    true,
+			},
+			"profile_versions": {
+				Type:        schema.TypeList,
+				Required:    true,
+				Description: "List of profile versions",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					StateFunc: func(val interface{}) string {
+						return strings.TrimSpace(val.(string))
+					},
+				},
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The type of the data export rule",
+				Default:     "Dataexport",
+			},
```

Comment:
why are we exposing this argument? the users can't change it, we should move this to be a computed argument or remove it completely.

Think about the `AWS_CFG` integration type, we created a resource specifically for that type and the user can't change it, if the type changes, must likely we will create a new resource.